alvr/client_core/resources/staging_fragment.glsl

```diff
@@ -3,9 +3,17 @@
 
 uniform samplerExternalOES tex;
 
+// Convert from limited colors to full
+const float LIMITED_MIN = 0.06274509803; // 16.0 / 255.0
+const float LIMITED_MAX = 0.92156862745; // 235.0 / 255.0
```

Comment:
We should restore the explicit fractions (in the comments). The decimal numbers were used because wgpu doesn't support expressions for constants.

Reply:
should be set now